src/pages/Route/components/Step1/RequestConfigView.tsx

```diff
@@ -47,7 +47,6 @@ const RequestConfigView: React.FC<Props> = ({ data, disabled, onChange }) => {
               <Form.Item
                 {...(index === 0 ? FORM_ITEM_LAYOUT : FORM_ITEM_WITHOUT_LABEL)}
                 label={index === 0 ? formatMessage({ id: 'route.request.config.domain.name' }) : ''}
-                required
```

Comment:
I found that remove `required` here only removed the `*`in the UI, 
in order to really remove the required verification of the host form item,  the required rules from line 59 to 63 must be removed as well :). @LiteSun @juzhiyuan

Reply:
DONE by @LiteSun